### ISSUE TITLE
make all failed on Mac 10.15.6

### ISSUE THREAD

**Issue by SeekPoint** [RedisGears/RedisGears#390]
17 warnings generated.
Packing dependencies...
Created artifacts/release/redisgears-dependencies.macosx-catalina-x64.99.99.99.tgz
Created artifacts/snapshot/redisgears-dependencies.macosx-catalina-x64.master.tgz
Linking bin/macosx-x64-release/redisgears.so...
Packing module...
Failed to pack release
/usr/local/opt/python@3.8/bin/python3.8: Error while finding module specification for 'RAMP.ramp' (ModuleNotFoundError: No module named 'RAMP')
make: *** [artifacts/release/redisgears.macosx-catalina-x64.99.99.99.zip] Error 1
redisLab-ghSrc/RedisGears %

**Comment by scuml** [comment]
Same error here.

**Comment by MeirShpilraien** [comment]
@rafie any idea how to fix it?

**Comment by rafie** [comment]
@MeirShpilraien This (or something similar) was once fixed in #302. I'll investigate.

**Comment by MeirShpilraien** [comment]
@rafie did your recent fixes will fix this issue?

**Comment by rafie** [comment]
@MeirShpilraien Yes, but I'd like to give it a little time before we declare it "solved". There are similar issues on other modules (e.g. RediSearch) and I'd like to see we catch all the fish.

**Comment by MeirShpilraien** [comment]
@rafie ^^

**Comment by Rambatino** [comment]
see: https://github.com/RedisGears/RedisGears/issues/464
